backend/tracim_backend/models/data.py

```diff
@@ -716,11 +723,17 @@ def copy(
 
         # copy attached_file
         if revision.depot_file:
-            copy_rev.depot_file = FileIntent(
-                revision.depot_file.file.read(),
-                revision.file_name,
-                revision.file_mimetype,
-            )
+            try:
+                copy_rev.depot_file = FileIntent(
+                    revision.depot_file.file.read(),
+                    revision.file_name,
+                    revision.file_mimetype,
+                )
+            except IOError as exc:
+                raise NewRevisionAbortedDepotCorrupted(
+                    "Can't create new revision of content if"
+                    " last revision file is not accessible."
```

Comment:
I suggest not to interpret the reason for IOError. (in most cases, "last revision file is not accessible." is the right interpretation, but you can't be sure - you are not the developer of FileIntent. I suggest you to reformulate like "Can't create new revision by copying another one. May be related to original revision file not being available"

Reply:
done